diff --git a/node_modules/expo-quest-notifications/android/build.gradle b/node_modules/expo-quest-notifications/android/build.gradle
index ae6a210..66da8cd 100644
--- a/node_modules/expo-quest-notifications/android/build.gradle
+++ b/node_modules/expo-quest-notifications/android/build.gradle
@@ -42,7 +42,7 @@ android {
 }
 
 dependencies {
-  implementation 'androidx.core:core:1.17.0'
+  implementation 'androidx.core:core:1.6.0'
   implementation 'androidx.lifecycle:lifecycle-runtime:2.9.3'
   implementation 'androidx.lifecycle:lifecycle-process:2.9.3'
   implementation 'androidx.lifecycle:lifecycle-common-java8:2.9.3'
